packages/generator-joplin/generators/app/templates/webpack.config.js

```diff
@@ -67,10 +68,18 @@ function currentGitInfo() {
 	}
 }
 
+function validateCategories(categories) {
+	if (!(categories.length === new Set(categories).size)) throw new Error('Repeated categories are not allowed');
+	categories.forEach(category => {
+		if (!allPossibleCategories.includes(category)) throw new Error(`${category} is not a valid category. Valid Categories are: \n${allPossibleCategories}\n`);
```

Comment:
As an error message, add `please make sure that the category name is lowercase`

Reply:
Added.